test/Bound/bound.lean

```diff
@@ -0,0 +1,132 @@
+/-
+Copyright (c) 2024 Geoffrey Irving. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Geoffrey Irving
+-/
+
+import Mathlib.Analysis.SpecialFunctions.Pow.Real
+import Mathlib.Tactic.Bound
+
+/-!
+## Tests for the `bound` tactic
+-/
```

Comment:
You've already mentioned that `bound` has scope-overlap with `positivity` and `gcongr`. It would be nice if, in the test file, you separate out tests which should only succeed with `bound`, but neither of the others. You can leave in some which overlap, I suppose, but it would be nice to have some clear demarcation.

Reply:
Will do.

Reply:
Done.